FsAutoComplete/CompilerServiceInterface.fs

```diff
@@ -0,0 +1,159 @@
+namespace FsAutoComplete
+
+open System
+open System.IO
+open Microsoft.FSharp.Compiler
+open Microsoft.FSharp.Compiler.SourceCodeServices
+
+type ParseAndCheckResults(parse: FSharpParseFileResults,
+                          check: FSharpCheckFileResults,
+                          version: int) =
+  let parseResults = parse
+  let checkResults = check
+  let _version = version
+
+  member x.TryGetMethodOverrides (lines: string[]) (line: int) (col: int) =
+    // Find the starting point, ideally right after the first '('
+    let lineCutoff = line - 3
```

Comment:
Is the `3` here saying to only look for the last 3 lines? That might not be enough if you have something like:

    Http.Request
         ( "http://some-url-here-that-is-very-long",
           query = ["page", "1"],
           ``meth`` = Get,
           someMoreHttpStuff =  ... )

Perhaps not something that needs to be fixed here, but maybe we can open separate issue for the future?